lib/server/db/model/Connection.js

```diff
@@ -0,0 +1,38 @@
+var mongoose = require("mongoose");
+
+/**
+ * Validates that a provided identifier is a positive number
+ *
+ * @private
+ * @function validateUserId
+ * @param {Number} id - A User's unique identifier
+ * @returns {Boolean}
+ */
+function validateUserId(id) {
```

Comment:
I think we can just remove it entirely for now. If we come back to this class later and decide that we want/need to add this function, we can add it back.